app/controllers/concerns/decidim/kids/has_minors_permissions.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Kids
+    module HasMinorsPermissions
```

Comment:
```suggestion
    module HasDecidimKidsPermissions
```
as this is not exclusive for the minors, maybe a more generic name would be better